plugin/Field/Collection/FieldsCollectionToContentFieldLabelArrayConverter.php

```diff
@@ -31,6 +31,8 @@
 
 class FieldsCollectionToContentFieldLabelArrayConverter
 {
+
+	private $_showMessageField = true;
```

Comment:
This class needs to stay stateless, and I don't think this is in use at all, so please remove this